package.json

```diff
@@ -49,7 +49,8 @@
     "jsforce": "^1.10.1",
     "jsonwebtoken": "8.5.0",
     "mkdirp": "1.0.4",
-    "sfdx-faye": "^1.0.9"
+    "sfdx-faye": "^1.0.9",
+    "ts-retry-promise": "^0.6.0"
```

Comment:
Is this 3PP approved?

Reply:
not yet.  Wanted to get agreement on the change first.

Reply:
I got that `retry` library approved overnight, might as well submit it now

Reply:
@mshanemc @WillieRuemmele I checked 3pp this morning and I could not find ts-retry-promise in the approved list. However, the [p-retry](https://www.npmjs.com/package/p-retry) module that Shane and I discuss on slack has already been approved. I would like to add that p-retry has millions of weekly downloads vs the relatively new ts-retry-promise, which has weekly activity < 100k.

Reply:
approved, thanks to Willie's magic touch.